content/en/releases/version-skew-policy.md

```diff
@@ -217,7 +217,7 @@ Optionally upgrade `kubelet` instances to **{{< skew currentVersion >}}** (or th
 
 {{< note >}}
 Before performing a minor version `kubelet` upgrade, [drain](/docs/tasks/administer-cluster/safely-drain-node/) pods from that node.
-In-place minor version `kubelet` upgrades are not supported.
+In-place minor version `kubelet` upgrades, to a newer minor version of Kubernetes, without a node drain, are not supported.
```

Comment:
I think the "to a newer minor version of Kubernetes" bit is redundant.

I also think the content of kubelet-managed directories is not necessarily cross-minor-version compatible. I'd like sig-node's input on highlighting the need to clear those directories before an in-place upgrade, something like this:

```suggestion
In-place minor version `kubelet` upgrades without draining pods and clearing the content of the kubelet-managed directories are not supported.
```

Reply:
OK, so combining the thinking so far, I suggest:
```suggestion
In-place `kubelet` upgrades, without first draining pods from the node **and** clearing the content
of the kubelet-managed directories, are only supported for upgrades to a newer patch release within
the same minor version of Kubernetes.
```

This makes for a problem, though: can we hyperlink “clearing the content of the kubelet-managed directories” to the specific text about how to do that? If not, we're almost certainly going to get issues filed about adding more detail.

Reply:
More information about this clearing of directories would definitely be good because the current documentation doesn't state anything about needing to do this - see https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes/

Reply:
@liggitt Is there somebody from sig-node you'd recommend asking for a consult on this?

Reply:
probably folks in https://github.com/kubernetes/community/tree/master/sig-node#technical-leads or delegates

cc @kubernetes/sig-node-pr-reviews
/cc @dchen1107 @derekwaynecarr @mrunalp

Reply:
Any chance one of you could weigh in here? @dchen1107 @derekwaynecarr @mrunalp

Reply:
/assign @mrunalp @dchen1107 @derekwaynecarr

Reply:
@mrunalp @dchen1107 @derekwaynecarr anything I can do to help move this PR forward?

Reply:
Best place for a list of files that the kubelet uses: within https://kubernetes.io/docs/reference/node/

Note that many of these are defaults or conventions; eg the kubelet even used to be able to fetch static Pod manifests via HTTP, `/etc/kubernetes/manifests` is just the most common place to put them.

Reply:
old thread on this topic:
https://github.com/kubernetes/website/issues/12326#issuecomment-704469840
but clearing contents on kubelet managed directories was never mentioned before?

> I think the ask here is to come up with the docs page on what files kubelet may be using.

CPU checkpoint files too maybe?
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/state/checkpoint.go#L31-L33

Reply:
https://github.com/kubernetes/website/pull/47425